src/models/galp6/README.md

```diff
@@ -0,0 +1,57 @@
+# Galago Pro (galp6)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Galago Pro](./img/galp6.webp)
+
+The System76 Galago Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 12th Generation (Alder Lake-U) CPUs
+        - [Core i7-1260P](https://ark.intel.com/content/www/us/en/ark/products/226254/intel-core-i71260p-processor-18m-cache-up-to-4-70-ghz.html)
+        - [Core i5-1240P](https://ark.intel.com/content/www/us/en/ark/products/132221/intel-core-i51240p-processor-12m-cache-up-to-4-40-ghz.html)
+- BIOS
+    - Macronix MX25L25673GZ4I-08G flash chip
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - Intel Iris Xe Graphics
+    - eDP display: 14.1" 1920x1080@60Hz LCD
+        - LCD panel: Panda LM140LF2L02 (or equivalent)
+    - External video outputs:
+        - 1x HDMI 2.1
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 64 (2x32GB) dual-channel DDR4 SO-DIMMs @ 3200 MHz
+- Networking
+    - Gigabit Ethernet
+    - M.2 PCIe/CNVi WiFi/Bluetooth
+        - Intel Wi-Fi 6 AX200/AX201
+- Power
+    - 90W (19V, 4.74A) DC-in port
+        - Barrel size: 5.5mm (outer), 2.5mm (inner)
+    - Included AC adapter: Chicony A16-090P1A
+        - AC power cord type: IEC C5
+    - USB-C charging compatible with 65W+ charger
```

Comment:
Should this mention a 90W+ charger since that's the wattage of the barrel charger? I'm sure a 65W will charge it, but I'm not sure if overdraw will make a charger that's <90W kinda unreliable for powering the machine under load.

Reply:
Fixed with [43fc99a](https://github.com/system76/tech-docs/pull/142/commits/43fc99a6b3921f341f41dd3b4f70298427218ddf).